src/browser.js

```diff
@@ -9,20 +9,24 @@
 /* eslint-env browser */
 
 import {createPlugin, type Context} from 'fusion-core';
+import {UniversalEventsToken} from 'fusion-plugin-universal-events';
 import {FetchToken} from 'fusion-tokens';
 import type {Fetch} from 'fusion-tokens';
 
 import type {HandlerType} from './tokens.js';
 import type {RPCPluginType, IEmitter} from './types.js';
 
+const statKey = 'rpc:method-client';
+
 class RPC {
   ctx: ?Context;
   emitter: ?IEmitter;
   handlers: ?HandlerType;
   fetch: ?Fetch;
 
-  constructor(fetch: Fetch) {
+  constructor(fetch: Fetch, emitter: any) {
```

Comment:
Should the type for `emitter` by `[IEmitter`](https://github.com/fusionjs/fusion-plugin-universal-events/blob/master/src/types.js#L15) here?

Reply:
Good catch, I had copied this from the server implementation, updated both